torch/nn/parallel/distributed.py

```diff
@@ -528,6 +528,11 @@ class DistributedDataParallel(Module, Joinable):
                          >>> ...
                          >>> ddp_logging_data = model_DDP._get_ddp_logging_data()
                          >>> static_graph = ddp_logging_data.get("can_set_static_graph")
+        delay_all_reduce_named_params (list of tuple of str and torch.nn.Parameter): a list
+                    of named parameters whose all reduce will be delayed into the backward
```

Comment:
nit: will be delayed when the gradient of the parameter specified in ``param_to_hook_all_reduce`` is ready

Reply:
nit: could we explain a little bit about how the two arguments will work with other arguments?

Reply:
Thanks Yanli for suggestions! I revised comments to improve these.